spec/models/like_spec.rb

```diff
@@ -1,17 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe Like, type: :model do
-  subject { Like.new(author_id: '1', posts_id: '2') }
-
-  before { subject.save }
-
-  it 'author id should be present' do
-    subject.author_id = nil
-    expect(subject).to_not be_valid
-  end
-
-  it 'post id should be present' do
-    subject.posts_id = nil
-    expect(subject).to_not be_valid
-  end
```

Comment:
- [ ] Kindly make sure to wrap all your tests in a `describe` statement, so your tests are grouped
- [ ] Also make sure to Create Unit tests for your `Like` model, they got removed after you revert changes to this commit